src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -220,7 +220,10 @@ class RideConstructionWindow final : public Window
         show_gridlines();
 
         _currentTrackPrice = MONEY32_UNDEFINED;
-        _currentBrakeSpeed2 = 8;
+#ifndef UNIFIED_BRAKE_SPEED_ENABLED
+        _currentBrakeSpeed2Temp = 3;
```

Comment:
2Temp is an index value that represents the speed in _allowedBrakeSpeedsTemp array. It is used to limit the valid speeds to what .td6 can support.